app/scripts/directives/logViewer.js

```diff
@@ -427,7 +427,7 @@ angular.module('openshiftConsole')
                                         podname: name,
                                         containername: $scope.options.container,
                                         backlink: URI.encode($window.location.href)
-                                      }))
+                                      }, $scope.context.project.metadata.annotations['openshift.io/logging.data.prefix']))
```

Comment:
This will be a runtime error if the project doesn't have any annotations. Better to use the annotation filter.

```
$filter('annotation')($scope.context.project, 'openshift.io/logging.data.prefix')))
```